docs/resources/samples/macros-power-automate.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Tip for how to use macro files or xlsm files in Power Automate flows'
+description: 'Learn how to use macro files or xlsm files in Power Automate flows.'
+ms.date: 03/18/2021
+localization_priority: Normal
+---
+
+# Tip for how to use macro files or xlsm files in Power Automate flows
```

Comment:
This doesn't seem like a sample. We should find a different home for this.

Reply:
Yeah, I don't have it in the TOC yet. Tracking with [feature](https://office.visualstudio.com/OC/_workitems/edit/4924924)